code/_expts/slotnick_etal_2001/analyze/plots/de_FigurizerCC.m

```diff
@@ -5,36 +5,28 @@
 % squares etc)
 
 ds = 'test';
+taskType = strcat(upper(mSets.data.taskType(1)), mSets.data.taskType(2:end));
+
 
 left = stats.rej.cc.perf.(ds){1};
 right = stats.rej.cc.perf.(ds){2};
 
-if strcmp(mSets.data.taskType, 'categorical') 
-    
-    left_results = left{1};
-    right_results = right{1};
-    
-    left_average = mean(left_results(:)');
-    right_average = mean(right_results(:)');
-    
-    left_stddev = std(left_results(:)');
-    right_stddev = std(right_results(:)');
-    fig = de_CreateSlotnickFigure1([left_average right_average], [left_stddev right_stddev], 'Categorical');
-    figs = [fig];
+left_results1 = left{1};
+right_results1 = right{1};
```

Comment:
Can you add a comment to what `results1` and `results2` are? (on/off or near/far)